code/game/objects/items/hfblade.dm

```diff
@@ -0,0 +1,142 @@
+// Chaplains begone!
+/obj/item/nullrod/scythe/vibro
+	name = "high fronquency bladd"
+
+/obj/item/melee/hfblade
+	name = "high frequency blade"
+	desc = "RULES OF NATURE"
+	icon_state = "hfblade"
+	item_state = "hfblade"
+	icon = 'icons/obj/hfblade.dmi'
+	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	flags_1 = CONDUCT_1
+	obj_flags = UNIQUE_RENAME
+	force = 0.001
+	throwforce = 20
+	w_class = WEIGHT_CLASS_BULKY
+	block_chance = 50
+	armour_penetration = 75
+	sharpness = IS_SHARP
+	attack_verb = list("chopped", "sliced", "cut", "zandatsu'd")
+	hitsound = 'sound/weapons/rapierhit.ogg'
+	materials = list(MAT_METAL = 10000)
+	light_color = "#40ceff"
+	var/rules_of_nature = TRUE // Turn this off to break the rules, and watch the horrors of no proximity flags unfold.
+	var/brazil = FALSE
+	var/brightness = 5
+
+/obj/item/melee/hfblade/Initialize()
+	. = ..()
+	AddComponent(/datum/component/butchering, 30, 95, 5)
+	set_light(brightness)
+	START_PROCESSING(SSobj, src)
+	. = ..()
+
+/obj/item/melee/hfblade/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+
+/obj/item/melee/hfblade/hit_reaction(mob/living/carbon/human/owner, mob/living/carbon/human/attacker, datum/martial_art/attacker_style, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(attack_type == PROJECTILE_ATTACK)
+		final_block_chance = 0
+	if(attack_type == MELEE_ATTACK) // It really is not a good idea to get close to this thing.
+		final_block_chance = 100
+		var/hideandseeklogic = rand(1,5) // YES, ADD MORE RNG TO COMBAT.
+		if(hideandseeklogic == 1 || HAS_TRAIT(attacker, TRAIT_CLUMSY))
+			attacker.visible_message("<span class = 'warning'>[attacker] attempts to attack [owner] and accidentally slices their arm off! What an idiot!</span>")
+			var/which_hand = BODY_ZONE_L_ARM
```

Comment:
These 3 lines can be simplified to `var/which_hand = (attacker.active_hand_index % 2) ? BODY_ZONE_L_ARM : BODY_ZONE_R_ARM`

Reply:
Changed, thanks for the input.